src/robusta/runner/config_loader.py

```diff
@@ -105,6 +106,7 @@ def __load_playbooks_repos(
         playbooks_repos: Dict[str, PlaybookRepo],
     ):
         playbook_packages = []
+        GitRepoManager.setup_host_keys(CUSTOM_SSH_HOST_KEYS.split("\n"))
```

Comment:
I think create a change of behavior I'm a little concerned with
 
Up until now, writing the known_hosts `/root` was happening only for user that defined a `git_key`
With this change it happens to all users

I know, that on some clusters we have limited access to disk, and might not have access to `/root/`
This will fail for them when the runner starts

Can we change it, to run once, if there actually `git_key` configured?